packages/core/docker/entrypoint.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+set -e
+
+# Run database migrations
+pnpm db:migrate
+
+cleanup() {
+  echo "Core container stopped"
+}
+
+# Check if NODE_ENV is set to development.
+if [ "$NODE_ENV" = "development" ]; then
```

Comment:
This chunk is here to keep alive core container in development to be able to connect to port 3002 to Drizzle studio. AI is awesome